Assets/Scripts/Models/Character/Character.cs

```diff
@@ -425,7 +429,7 @@ public string GetDescription()
 
     public IEnumerable<string> GetAdditionalInfo()
     {
-        yield return string.Format("HitPoints: 100/100");
+        yield return string.Format("HitPoints: " + currentHP + "/" + maxHP);
```

Comment:
Additionally there's also a localization string for this, though I don't know if the info window currently uses a TextLocalizer or how the TextLocalizer will handle larger conglomerate texts such as those used in the info window.